lib/bundler/templates/newgem/travis.yml.tt

```diff
@@ -1,7 +1,5 @@
 ---
-sudo: false
 language: ruby
 cache: bundler
-rvm:
-  - <%= RUBY_VERSION %>
```

Comment:
I don't agree with this change. It doesn't offer any benefit, and we're adding more steps for users that want to test multiple ruby versions

Reply:
Yeah, I agree with @colby-swandale here actually. The current configuration (aesthetics appart) was great for a single ruby too, so should work for every one. Let's leave it as it was, with the added benefit that you don't need to fix the failing spec :)

Reply:
There's a majority, I'll revert this bit - thanks for looking at it.